pom.xml

```diff
@@ -294,8 +294,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>${compiler.plugin.verson}</version>
                 <configuration>
-                    <source>${java.version}</source>
-                    <target>${java.version}</target>
+                    <source>8</source>
+                    <target>8</target>
```

Comment:
Given that we released Koryphe 2.0.0 as Java 11 only, wouldn't it be better if the jar without a classifier was Java 11?